OpenTabletDriver.Desktop/Diagnostics/DiagnosticInfo.cs

```diff
@@ -23,11 +24,34 @@ EnvironmentDictionary environmentDictionary
 
         public string AppVersion { get; } = GetAppVersion();
         public string BuildDate { get; } = typeof(BuildDateAttribute).Assembly.GetCustomAttribute<BuildDateAttribute>()?.BuildDate ?? string.Empty;
-        public OperatingSystem OperatingSystem { get; } = Environment.OSVersion;
+        public IDictionary<string, string> OperatingSystem
+        {
+            get
+            {
+                switch (SystemInterop.CurrentPlatform)
+                {
+                    case SystemPlatform.Linux:
+                        return LinuxOSRelease.Raw;
+                    default:
+                        return OSVersionToDict();
+                }
+            }
+        }
```

Comment:
```suggestion
        public IDictionary<string, string> OperatingSystem => SystemInterop.CurrentPlatform switch
        {
            SystemPlatform.Linux => LinuxOSRelease.Raw,
            _ => OSVersionToDict(),
        };
```

@X9VoiD reminded me that this exists. Didn't test but should compile/work?

Reply:
Will fixup if it doesnt work, looks familiar to me